apps/dapp-console-api/src/routes/auth/AuthRoute.spec.ts

```diff
@@ -11,30 +12,113 @@ import {
 import { Trpc } from '@/Trpc'
 
 import { AuthRoute } from './AuthRoute'
+import { envVars } from '@/constants'
 
 describe(AuthRoute.name, () => {
   let trpc: Trpc
   let caller: ReturnType<RouterCaller<AuthRoute['handler']['_def']>>
-  const session = mockUserSession({
-    entityId: 'id1',
-    privyAccessTokenExpiration: Date.now(),
-    privyAccessToken: 'accessToken',
-    privyDid: 'privy:did',
-  })
+  let badCaller: ReturnType<RouterCaller<AuthRoute['handler']['_def']>>
+  const validSession = mockUserSession(
+    {
+      entityId: 'id1',
+      privyAccessTokenExpiration: Date.now(),
+      privyAccessToken: 'accessToken',
```

Comment:
this expiration and privyAccessToken will not pass validation. The expiration is set to now, so it will be counted as expired and the accessToken is not hashed properly. I would consider extending the `validSession` helper to allow you to pass in the world id session: https://github.com/ethereum-optimism/ecosystem/blob/cedaa83b206f1379624c9b395c6b6114323fd8b0/apps/dapp-console-api/src/testhelpers/auth.ts#L63